MAUI/Scheduler/load-on-demand.md

```diff
@@ -0,0 +1,311 @@
+---
+layout: post
+title: Load appointments on demand in .NET MAUI Scheduler | Syncfusion
+description: Learn here all about to load appointments on demand from visible dates in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more. 
```

Comment:
Learn here all about how to load appointments on demand from visible dates in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more.